login.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+require_once 'src/connectToDb.php';
+require_once 'src/Models/UserModel.php';
+
+session_start();
+
+$_SESSION['userid']= false;
+$db = connectToDb();
+$userModel = new UserModel($db);
+$errorMessage = '';
+
+if(isset($_POST['submit'])) {
+    $inputtedUsername = $_POST['username'];
+    $inputtedPassword = $_POST['password'];
+
+    $verifyUser = $userModel->getUserByUsername($inputtedUsername);
+    if($verifyUser === false) {
+        $errorMessage = 'Invalid username or password';
+    }elseif (password_verify($inputtedPassword, $verifyUser->password)){
+        $_SESSION['userid']=$verifyUser->id;
+        header('Location: index.php');
+    }else {
+        $errorMessage = 'Invalid username or password';
+    }
+}
+
+if ($_SESSION['userid']===false){
+    $isUserLoggedIn = '<a href="login.php">Login</a>
+        <a href="#">Register</a>';
+} else {
+    $isUserLoggedIn = '<a href="#">Create Post</a>';
+}
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Blog - Login</title>
+    <script src="https://cdn.tailwindcss.com"></script>
+</head>
+<body class="selection:bg-teal-200">
+<nav class="flex justify-between items-center py-5 px-4 mb-10 border-b border-solid">
+    <a href="index.php"><h1 class="text-5xl">Blog</h1></a>
+    <div class="flex gap-5">
+
+        <?php echo $isUserLoggedIn; ?>
+    </div>
+</nav>
+
+<form method="post" class="container lg:w-1/4 mx-auto flex flex-col p-8 bg-slate-200">
+    <h2 class="text-3xl mb-4 text-center">Login</h2>
+    <div class="mb-5">
+        <label class="mb-3 block" for="username">Username:</label>
+        <input class="w-full px-3 py-2 text-lg" type="text" id="username" name="username" />
```

Comment:
There is a mistake here - which is completely my fault. The story specifies that email address should be used to login, rather than username.

My preference would be that you swap over to email and update accordingly, how much work do you think that would be?